### DISCUSSION TITLE
Snowpack and esbuild?

### DISCUSSION THREAD

**Discussion by mindplay-dk** [JSMonk/hegel#329] [Ideas]
Have you tried to use Hegel with next-gen build tools like esbuild or Snowpack?

It would be great if we could get some setup instructions for newer tools into the documentation. If they work with TS and Flow, they should theoretically work with Hegel as well, at least in terms of syntax, right?

I might give esbuild a try myself and try to get a workable setup soon. I'll post my results. 🙂

**Comment by JSMonk** [comment]
No, we didn't try. But sounds like an interesting experiment.
So, we will try 😸

**Comment by thecotne** [comment]
esbuild/Snowpack setup should be same as for flow (since hegel uses flow syntax)
